### PR TITLE
[prometheus-elasticsearch-exporter] Add topologySpreadConstraints

### PR DESCRIPTION
**Pull request by m-yosefpor**
#### What this PR does / why we need it
Add `topologySpreadConstraints` for the deployment

#### Checklist
* [x]  [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
* [x]  Chart Version bumped
* [x]  Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-elasticsearch-exporter/values.yaml [hunk 1]
```diff
@@ -94,6 +94,8 @@ commonLabels: {}
 
 affinity: {}
 
+topologySpreadConstraints: {}
```
Comment (zeritti) [inline charts/prometheus-elasticsearch-exporter/values.yaml:97]: `spec.topologySpreadConstraints` is an array:

```suggestion
topologySpreadConstraints: []
```

**Review by zeritti — CHANGES_REQUESTED** [review]
Thank you, @m-yosefpor, for your PR. Please, see my comments below.

**Comment by m-yosefpor** [comment]
> Thank you, @m-yosefpor, for your PR. Please, see my comments below.

Done. Thanks for your attention.

**Review by zeritti — APPROVED** [review]
Thank you, LGTM.

**Pull request merged**